src/sqlparser/src/ast/mod.rs

```diff
@@ -17,6 +17,7 @@ mod legacy_source;
 mod operator;
 mod query;
 mod statement;
+pub mod utils;
```

Comment:
What about giving it a more reasonable name?

Reply:
I've considered moving it out of the sqlparser crate since these serializers are not commonly used. What about moving it to the frontend crate and renaming it to redact.rs (more related functions will be added there)?

Reply:
Shall we continue this PR? @BugenZhao

Reply:
> What about moving it to the frontend crate and renaming it to redact.rs (more related functions will be added there)?

LGTM